Find in Project keybind missing

### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Find in Project keybind will not be displayed at the top. The keybind works, it just isn't displaying the keybind

Shouldn't Find & Find in Project be grouped under `Go`. Replace would belong under Edit, but I don't get why Find

### Environment

Zed: v0.130.7 (Zed)
OS: macOS 14.0.0
Memory: 16 GiB
Architecture: aarch64

### If applicable, add mockups / screenshots to help explain present your vision of the feature

<img width="226" alt="Screenshot 2024-04-16 at 14 33 29" src="https://github.com/zed-industries/zed/assets/121750393/671ab603-1204-45b4-b149-82cb6b3af438">


### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.

2024-04-16T14:37:05+02:00 [INFO] ========== starting zed ==========
2024-04-16T14:37:05+02:00 [INFO] Opening main db
2024-04-16T14:37:05+02:00 [INFO] Using git binary path: Some("/Applications/Zed.app/Contents/MacOS/git")
2024-04-16T14:37:06+02:00 [ERROR] crates/theme/src/settings.rs:371: theme not found: Zedokai Darker (Filter Spectrum)
2024-04-16T14:37:06+02:00 [INFO] extensions updated. loading 21, reloading 0, unloading 0
2024-04-16T14:37:06+02:00 [INFO] set environment variables from shell:/bin/zsh, path:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Users/frederik/.cargo/bin:/Users/frederik/Library/Application Support/JetBrains/Toolbox/scripts
2024-04-16T14:37:06+02:00 [ERROR] crates/zed/src/zed.rs:631: EOF while parsing a value at line 1 column 0
2024-04-16T14:37:06+02:00 [INFO] Opening main db
2024-04-16T14:37:06+02:00 [INFO] building git repository, `.git` path in the worktree: ".git"
2024-04-16T14:37:06+02:00 [ERROR] crates/fs/src/repository.rs:122: Error { code: -9, klass: 4, message: "reference 'refs/heads/master' not found" }
2024-04-16T14:37:06+02:00 [INFO] Opening main db
2024-04-16T14:37:06+02:00 [INFO] set status on client 0: Authenticating
2024-04-16T14:37:06+02:00 [INFO] Opening main db
2024-04-16T14:37:06+02:00 [INFO] set status on client 119171: Connecting
2024-04-16T14:37:06+02:00 [INFO] starting language server "rust-analyzer", path: "/Users/frederik/code/rust/test1", id: 1
2024-04-16T14:37:06+02:00 [INFO] fetching latest version of language server "rust-analyzer"
2024-04-16T14:37:06+02:00 [INFO] reload git repository ".git"
2024-04-16T14:37:06+02:00 [ERROR] crates/fs/src/repository.rs:122: Error { code: -9, klass: 4, message: "reference 'refs/heads/master' not found" }
2024-04-16T14:37:06+02:00 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
2024-04-16T14:37:07+02:00 [INFO] reload git repository ".git"
2024-04-16T14:37:07+02:00 [ERROR] crates/fs/src/repository.rs:122: Error { code: -9, klass: 4, message: "reference 'refs/heads/master' not found" }
2024-04-16T14:37:07+02:00 [INFO] downloading language server "rust-analyzer"
2024-04-16T14:37:07+02:00 [INFO] starting language server. binary path: "/Users/frederik/Library/Application Support/Zed/languages/rust-analyzer/rust-analyzer-2024-04-15", working directory: "/Users/frederik/code/rust/test1", args: []
2024-04-16T14:37:07+02:00 [INFO] add connection to peer
2024-04-16T14:37:07+02:00 [INFO] waiting for server hello
2024-04-16T14:37:07+02:00 [INFO] got server hello
2024-04-16T14:37:07+02:00 [INFO] set status to connected (connection id: ConnectionId { owner_id: 0, id: 0 }, peer id: PeerId { owner_id: 392, id: 321468 })
2024-04-16T14:37:07+02:00 [INFO] set status on client 119171: Connected { peer_id: PeerId { owner_id: 392, id: 321468 }, connection_id: ConnectionId { owner_id: 0, id: 0 } }
2024-04-16T14:37:14+02:00 [ERROR] crates/lsp/src/lsp.rs:728: oneshot canceled

--- a/crates/zed/src/zed/app_menus.rs
+++ b/crates/zed/src/zed/app_menus.rs
@@ -68,7 +68,7 @@ pub fn app_menus() -> Vec<Menu<'static>> {
                 MenuItem::os_action("Paste", editor::actions::Paste, OsAction::Paste),
                 MenuItem::separator(),
                 MenuItem::action("Find", search::buffer_search::Deploy::find()),
-                MenuItem::action("Find In Project", workspace::NewSearch),
+                MenuItem::action("Find In Project", workspace::DeploySearch::find()),
                 MenuItem::separator(),
                 MenuItem::action(
                     "Toggle Line Comment",

